src/transformers/models/mobilebert/tokenization_mobilebert.py

```diff
@@ -33,17 +39,466 @@
 PRETRAINED_INIT_CONFIGURATION = {}
 
 
-class MobileBertTokenizer(BertTokenizer):
+# Copied from transformers.models.bert.tokenization_bert.load_vocab
+def load_vocab(vocab_file):
+    """Loads a vocabulary file into a dictionary."""
+    vocab = collections.OrderedDict()
+    with open(vocab_file, "r", encoding="utf-8") as reader:
+        tokens = reader.readlines()
+    for index, token in enumerate(tokens):
+        token = token.rstrip("\n")
+        vocab[token] = index
+    return vocab
+
+
+# Copied from transformers.models.bert.tokenization_bert.whitespace_tokenize
+def whitespace_tokenize(text):
+    """Runs basic whitespace cleaning and splitting on a piece of text."""
+    text = text.strip()
+    if not text:
+        return []
+    tokens = text.split()
+    return tokens
+
+
+# Copied from transformers.models.bert.tokenization_bert.BertTokenizer with BERT->MobileBERT
```

Comment:
```suggestion
# Copied from transformers.models.bert.tokenization_bert.BertTokenizer with BERT->MobileBERT, Bert->MobileBert
```
You have on BertTokenizer in the body of the class (see comment below) that needs to be adapted.